xlators/mgmt/glusterd/src/glusterd-pmap.c

```diff
@@ -582,16 +281,14 @@ __gluster_pmap_signout(rpcsvc_request_t *req)
         gf_smsg(this->name, GF_LOG_ERROR, errno, GD_MSG_GARBAGE_ARGS, NULL);
         goto fail;
     }
-    rsp.op_ret = pmap_registry_remove(THIS, args.port, args.brick,
-                                      GF_PMAP_PORT_BRICKSERVER, req->trans,
-                                      _gf_false);
+
+    rsp.op_ret = 0;
 
     ret = glusterd_get_brickinfo(THIS, args.brick, args.port, &brickinfo);
     if (args.rdma_port) {
         snprintf(brick_path, PATH_MAX, "%s.rdma", args.brick);
-        rsp.op_ret = pmap_registry_remove(THIS, args.rdma_port, brick_path,
-                                          GF_PMAP_PORT_BRICKSERVER, req->trans,
-                                          _gf_false);
+
+        rsp.op_ret = 0;
```

Comment:
I don't think we still support RDMA, do we? I think we might be able to remove this part altogether. 
See https://review.gluster.org/#/c/glusterfs/+/24031/ (which we never merged)